apps/trade-tracker/.env.example

```
@@ -0,0 +1,10 @@
+# Convex
+# Get these from https://dashboard.convex.dev after creating a project
+CONVEX_DEPLOYMENT=dev:your-deployment-name
+NEXT_PUBLIC_CONVEX_URL=https://your-deployment-name.convex.cloud
+
+# Clerk Authentication
+# Get these from https://dashboard.clerk.com after creating an application
+NEXT_PUBLIC_CLERK_FRONTEND_API_URL=https://your-clerk-instance.clerk.accounts.dev
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_test_...
+CLERK_SECRET_KEY=sk_test_...
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CLERK_SECRET_KEY key should go before the NEXT_PUBLIC_CLERK_FRONTEND_API_URL key

(UnorderedKey)